chatbot/setup_postgresql_local.sh

```diff
@@ -0,0 +1,4 @@
+sudo -u postgres psql -p $PG_PORT -c "CREATE ROLE $PG_USER PASSWORD '$PG_PASSWORD' LOGIN"
+sudo -u postgres createdb -p $PG_PORT -O $PG_USER $PG_USER
+sudo -u postgres psql -p $PG_PORT -d $PG_USER -c 'CREATE EXTENSION IF NOT EXISTS "pgcrypto"'
+./setup_postgresql.sh $1 $2 $3
```

Comment:
What do the `$1 $2 $3` do here?

Reply:
we have a migration script that takes in parameters for adding a first installation and those are the parameters